### DISCUSSION TITLE
Multiple xenium data analysis

### DISCUSSION THREAD

**Discussion by ilovegreentea** [satijalab/seurat#8616] [General]
Hello..

I going to analyse mutiple xenium data set.
so i just read each Xenium data and merged with merge() function.
after that, i processed the data.

but i'm wondering how plotting ImageDimplot the data which I want to see.
and i have problem at subsetting data.

**Error messege is below**
> subset(merge, sample_name != 'yet')
Error in `as.Graph()`:
! Please provide rownames to the matrix before converting to a Graph
Run `rlang::last_trace()` to see where the error occurred.

who have any idea to analysis multiple xenium data set, please give me any advice.

thank you.

**Comment by RajneeshSrivastava** [comment]
Seurat Team, 
Just curious, if there is a solution for merging multiple xenium dataset, as issue raised by @ilovegreentea

**Reply by ilovegreentea** [reply]
I Found some solution below. Using subset_opt function

https://github.com/satijalab/seurat/issues/8848
https://github.com/satijalab/seurat/discussions/7947#discussioncomment-9500038

**Reply by ramosemi** [reply]
Hello Seurat team,
I would like to analyze 4 xenium samples and perform a pseudobulk analysis and compare expression of a few genes (PTX3 for example) in the fibroblast population of each sample. Do you have a general recommendation on how to perform pseudobulk analysis for multiple Xenium samples?

**Reply by swambha** [reply]
Any leads on performing pseudobulk on Xenium datasets?
